oemer/rhythm_extraction.py

```diff
@@ -109,7 +128,13 @@ def polish_symbols(staff_pred, symbols, stems, clefs_sfns, group_map):
     return beams
 
 
-def parse_beams(min_area_ratio=0.07, min_tp_ratio=0.4, min_width_ratio=0.2):
+def parse_beams(
+        min_area_ratio: float = 0.07, 
+        min_tp_ratio: float = 0.4, 
+        min_width_ratio: float = 0.2) -> Tuple[
+            ndarray, 
+            List[Tuple[Tuple[float, float], Tuple[float, float], float]], 
+            ndarray]:
```

Comment:
Formatting

Reply:
Not sure how you would format the Tuple. I joined it again in one line as then it stays in the 120 char limit you gave earlier and I haven't found any other Tuple which spanned multiple lines.

Reply:
Just for reference, no need to modify. In this case, if the line is over the max allowed length , I would reformat it like following:

```
def parse_beams(
        min_area_ratio: float = 0.07, 
        min_tp_ratio: float = 0.4, 
        min_width_ratio: float = 0.2
) -> Tuple[
    ndarray, List[Tuple[Tuple[float, float], Tuple[float, float], float]], ndarray
]:
```